core/shell/shell.go

```diff
@@ -144,7 +144,7 @@ func (s *DefaultShell) Exec(command builder.Command, extraArgs ...string) (outSt
 
 	cmd = execCmdFn(exe, args...)
 	cmd.Env = os.Environ()
-	cmd.Stdin = s.InStream()
+	// cmd.Stdin = s.InStream()
```

Comment:
why?

Reply:
Good catch, this is a bad test change that we must undo.